tasks/agent.py

```diff
@@ -577,29 +576,28 @@ def hacky_dev_image_build(
 
 
 @task
-def integration_tests(ctx, install_deps=False, race=False, remote_docker=False, go_mod="mod"):
+def integration_tests(ctx, race=False, remote_docker=False, go_mod="mod", timeout="10m"):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why is it 10m by default while it is 30m for both linux and windows in the gitlab config? (same for other parts of the code using this)

Reply:
Discussed offline: you're right that was off. Fixed with [bc30670](https://github.com/DataDog/datadog-agent/pull/29489/commits/bc306707fabdde1f660125262ca795019c743ee7) and [e45cd34](https://github.com/DataDog/datadog-agent/pull/29489/commits/e45cd341cc63a0bd66d298784f4ed8ef9a07730d).